docs/src/main/asciidoc/hibernate-search-elasticsearch.adoc

```diff
@@ -476,8 +476,8 @@ In our existing `LibraryResource`, we just need to inject the following methods
     @GET
     @Path("author/search") // <3>
     @Transactional
-    public List<Author> searchAuthors(@QueryParam String pattern, // <4>
-            @QueryParam Optional<Integer> size) {
+    public List<Author> searchAuthors(@QueryParam("pattern") String pattern, // <4>
+            @QueryParam("size") Optional<Integer> size) {
```

Comment:
This change shouldn't be needed, see the note: you should use `org.jboss.resteasy.annotations.jaxrs.QueryParam` for the import.

Reply:
Ok, sorry.